src/screens/Welcome.js

```diff
@@ -32,7 +32,7 @@ const Welcome = ({navigation}) => {
         <TouchableOpacity
           style={styles.button}
           onPress={() => {
-            console.log('Hello World');
+            navigation.navigate('RegisterCompleted');
```

Comment:
I assume that when you navigate into the `RegisterCompleted` screen, you navigate when the  user put his contact information like this screen not when the user touches the "Iniciar sesión" button 

![image](https://user-images.githubusercontent.com/42794917/96511551-d9f88e80-1224-11eb-8f97-4575d52dbcd8.png)